prclz/reblock/planar_graph.py

```diff
@@ -588,6 +579,38 @@ def add_node_to_closest_edge(self,
         # Now add it
         self.split_edge_by_node(closest_edge, closest_node, terminal=terminal)
 
+    def add_buildings(self: PlanarGraph, 
```

Comment:
I don't think you can use a type annotation for class within its own definition (the type doesn't exist when the method is defined, weirdly enough)

Reply:
Yeah I just add a dummy PlanarGraph = None to the top before (re)defining the class bc yes, I've noticed that too. It's sort of annoying to not be able to denote that class factory methods return an instance of the classs but such is life. Do ever do anything to enforce the type annotations?